_dashboards/quickstart-dashboards.md

```diff
@@ -11,9 +11,6 @@ redirect_from:
 
 This quickstart guide covers the core concepts that you need to understand to get started with OpenSearch Dashboards. You'll learn how to:
 
-- Add sample data.
-- Explore and inspect data with **Discover**.
-- Visualize data with **Dashboard**.
 - Add sample data.
 - Explore and inspect data with **Discover**.
```

Comment:
What is Discover exactly? Can't we just say Explore and inspect data?

Reply:
Discover is the application on the UI.

Reply:
change made